src/components/base/NavigationItem/NavigationItem.module.scss

```diff
@@ -0,0 +1,125 @@
+@use "../../../styles/utilities/functions" as *;
+@use "../../../styles/mixins/mixins-typo" as *;
+@use "../../../styles/utilities/mediaqueries" as *;
+@use "../../../styles/utilities/utilities" as *;
+@use "../../../styles/utilities/animations" as *;
+
+.expandableMenu {
+  --button-text-size: var(--font-size-m);
+
+  padding: 0;
+  cursor: pointer;
+  position: relative;
+
+  @include mq($from: large) {
+    padding-inline-start: 0;
+    padding-inline-end: 0;
+  }
+
+  .buttonWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: px-to-rem(var(--spacing-m));
+
+    & {
+      margin-block-end: px-to-rem(var(--spacing-s));
+    }
+
+    @include mq($from: large) {
+      margin-block-end: px-to-rem(var(--spacing-xl));
+    }
+
+    .buttonLevel1 {
```

Comment:
Tiny detail, I would probably set the buttons take up the entire width in the mobile nav. We can always fix that later.